### PR TITLE
Recording: with empty config, save default split size immediately

### PR DESCRIPTION
**Pull request by ronso0**
reported in the forum
https://mixxx.discourse.group/t/recording-cut-into-parts/29631

### CODE REVIEW

**Review by daschuer — CHANGES_REQUESTED** [review]
Thank you for the fix. Can you please also adjust this: 
https://github.com/mixxxdj/mixxx/blob/18a3e4a75f776969323b860828df1437e1dca534/src/recording/recordingmanager.cpp#L328

A common class for all the recording defaults would be also nice but probably out of the scope of this bug-fix PR:
"preferences/replaygainsettings.cpp"

**Comment by ronso0** [comment]
Oh, right.
Done.

File: src/preferences/dialog/dlgprefrecord.cpp [hunk 1]
```diff
@@ -92,10 +92,12 @@ DlgPrefRecord::DlgPrefRecord(QWidget* parent, UserSettingsPointer pConfig)
     if (index >= 0) {
         // Set file split size
         comboBoxSplitting->setCurrentIndex(index);
-    }
-    else {
+    } else {
         //Use max RIFF size (4GB) as default index, since usually people don't want to split.
         comboBoxSplitting->setCurrentIndex(4);
+        // Save the value immediately, otherwise RecordingManager would fall back
```
Comment (daschuer) [inline src/preferences/dialog/dlgprefrecord.cpp:98]: The comment is now outdated.

Comment (ronso0) [reply src/preferences/dialog/dlgprefrecord.cpp]: Yeah, well, I overlooked that we actually only need to use the same default in both unit.
Saving to the config doesn't hurt though.

**Review by daschuer — APPROVED** [review]
LGTM, Thank you.

**Pull request merged**